src/utilcode/util.cpp

```diff
@@ -1288,7 +1288,15 @@ int GetCurrentProcessCpuCount()
     // the case where there are more than 64 processors, so we will return a
```

Comment:
CC @Maoni0 - or is this comment outdated now and we can support more? Should we fix the windows code in that case too?